### PR TITLE
Fix for 2 DNS question bugs

### PR DESCRIPTION
**Pull request by kalinkrustev**
Fixed 2 DNS question bugs:

* It was replying regardless of the DNS question domain
* Debug message was wrong

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -273,6 +273,8 @@ DNSDiscovery.prototype._push = function (id, port, host, socket) {
 
 DNSDiscovery.prototype._onquestion = function (query, port, host, answers, multicast) {
   var domain = parseDomain(query.name)
+  
+  if (domain !=== this._domain) return
```
[the remark this answers is not in the document]
Comment (kalinkrustev) [reply index.js]: sorry, fixed it

**Comment by pfrazee** [comment]
Just tested discovery to hashbase and to a local dat cli process. 👍 Both worked fine.

**Comment by kalinkrustev** [comment]
Hi, I hope you did not forget about this fix.

**Comment by mafintosh** [comment]
6.2.3

**Pull request merged**